front/app/modules/commercial/project_folders/admin/containers/settings/index.tsx

```diff
@@ -99,8 +99,8 @@ const Data = adopt<DataProps, WithRouterProps>({
   ),
 });
 
-export default (inputProps: WithRouterProps) => (
+export default withRouter((inputProps) => (
   <Data {...inputProps}>
     {(dataProps) => <FolderSettingsWithHoCs {...inputProps} {...dataProps} />}
   </Data>
-);
+));
```

Comment:
You could as well use useParams from now. :-)

Reply:
withRouter is needed in this component to pass the ID from params into the ` <GetProjectFolder` data wrapper, so I think we can leave it as-is to avoid rewriting. All three children of the project_folders layout container work as of the most recent commit